### ISSUE TITLE
Non-string action types are reported `<UNDEFINED>`

### ISSUE THREAD

**Issue by ajcrites** [zalmoxisus/redux-devtools-extension#459]
You can use anything as an action type, but it seems like only strings are logged properly.

<img width="237" alt="screen shot 2018-02-21 at 3 31 30 pm" src="https://user-images.githubusercontent.com/1308273/36503790-531f7f6e-171c-11e8-9a39-41058b8501c4.png">

For example if you have `LOAD_USER = Symbol('load user');` and then `dispatch({ type: LOAD_USER })`, it will log the action as `<UNDEFINED>`.

The action type should be logged as `String(action.type)`.

**Comment by zalmoxisus** [comment]
Use `serialize` in parameter for that:
```js
const store = Redux.createStore(reducer, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__({
  serialize: true
}));
```

See [API docs](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#serialize) for more details.

I actually was trying to do the way you suggested in `redux-devtools-instrument`, but that was causing issues when recomputing actions.

**Comment by iamscottcox** [comment]
I'm having a similar issue, but some of my Symbol action types are being logged.

<img width="646" alt="screenshot 2018-08-06 11 50 24" src="https://user-images.githubusercontent.com/14204638/43712729-19196bba-996f-11e8-9914-7f76faf4335b.png">

We have passed serialize as an option:
![screenshot 2018-08-06 11 53 59](https://user-images.githubusercontent.com/14204638/43712838-70b259cc-996f-11e8-87c1-72443977e356.png)

Any ideas on what might cause this? :)

**Comment by CoderKwong** [comment]
I also have this problem.How to fix it?

**Comment by ajcrites** [comment]
IMO you should probably use strings for action types versus something like symbols. My understanding is that strings are serializable, but Symbols are not. The Redux docs specify this; I don’t think there’s an advantage to using Symbols over strings for action types.

**Comment by kymar** [comment]
I recently had this problem and fixed it using the `actionSanitizer` key on object passed into the extension middleware to specifically convert the symbol to a string, seems like this is a little overkill but works for now ?

```js
const store = createStore(
  rootReducer,
  window.__REDUX_DEVTOOLS_EXTENSION__ &&
    window.__REDUX_DEVTOOLS_EXTENSION__({
      actionSanitizer: action => ({
        ...action,
        type: action.type.toString()
      })
    })
);

```

**Comment by zalmoxisus** [comment]
It was a regression in `jsan`, it's downgraded now and fixed in `2.16`, which will land o Chrome Store in a day.
